pkgs/by-name/si/sirula/package.nix

```
@@ -1,21 +1,21 @@
 {
+  nix-update-script,
   lib,
   fetchFromGitHub,
   rustPlatform,
   pkg-config,
   gtk3,
   gtk-layer-shell,
 }:
-
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
```

```suggestion
rustPlatform.buildRustPackage rec {
```
Added back rec for src.

---

Related resources:
https://github.com/NixOS/nixpkgs/issues/277994
https://github.com/NixOS/nixpkgs/issues/208242
https://github.com/NixOS/nixpkgs/issues/103997